src/main/java/org/myrobotlab/service/config/ServoConfig.java

```diff
@@ -0,0 +1,26 @@
+package org.myrobotlab.service.config;
+
+public class ServoConfig extends Config {
```

Comment:
why not just extend ServiceConfig ?

Reply:
maybe ... depends on which structure we like

Reply:
the subclass can/should have full freedom to implement the config that fits that services needs.

Reply:
eh?   it does ...